Season2_Level-3/solution.md

```diff
@@ -0,0 +1,40 @@
+### This code is vulnerable to [XSS](https://portswigger.net/web-security/cross-site-scripting)
+
+Vulnerable Endpoint: `/getPlanetInfo`
+
+Exploit: 
+```js
+http://localhost:5000/getPlanetInfo?planet=<script>alert('XSS Attack')</script>
+``` 
+
```

Comment:
Good job! I think what is missing here is your proposed solution (the secure code).

You can also explain the reason that the new code is now secure and why.